### DISCUSSION TITLE
How to set the theme of xwayland cursor?

### DISCUSSION THREAD

**Discussion by fxzxmicah** [hyprwm/Hyprland#8196] [Q&A]
I have these in the configuration file, but they are not effective for the xwayland application.
```
env = XCURSOR_SIZE,24
env = XCURSOR_THEME,breeze_cursors
env = HYPRCURSOR_SIZE,24
env = HYPRCURSOR_THEME,breeze_cursors
```

**Comment by JLH42** [comment]
what worked for me is going to /usr/share/icons/default/index.theme and put this in the .theme file
```
[Icon Theme]
Inherits=Bibata-Modern-Classic
```

but instead of the bibata change it to the cursor that you want like the breeze_cursors
then reload the xwayland apps

**Reply by Jysve** [reply]
Works fine, from niri👍
